### PR TITLE
Fix bug when argument is null, request is not  sended

### PR DESCRIPTION
**Pull request by mustafaboleken**
https://github.com/ant-media/Ant-Media-Server/issues/4330

### CODE REVIEW

File: src/main/java/io/antmedia/console/rest/CommonRestService.java [hunk 1]
```diff
@@ -261,17 +262,30 @@ public boolean sendUserInfo(String email, String firstname, String lastname, Str
 
 			httpPost.setConfig(requestConfig);
 
+			// if any of them is null, then it will throw IllegalArgumentException
+			// so, be sure that none of them is null
```
Comment (mekya) [inline src/main/java/io/antmedia/console/rest/CommonRestService.java:266]: Are you sure that this comment is true?

**Review by mekya — CHANGES_REQUESTED** [review]
Please just take a look at my comment and let me know if it's correct

File: src/main/java/io/antmedia/console/rest/CommonRestService.java (hunk 1 above)
Comment (mustafaboleken) [reply src/main/java/io/antmedia/console/rest/CommonRestService.java:266]: it think comment wrong, maybe I should change it to "if args are null, set them to empty string"

Comment (mekya) [reply src/main/java/io/antmedia/console/rest/CommonRestService.java:266]: Yes, please

**Review by mekya — CHANGES_REQUESTED** [review]

File: src/main/java/io/antmedia/console/rest/CommonRestService.java (hunk 1 above)
Comment (mustafaboleken) [reply src/main/java/io/antmedia/console/rest/CommonRestService.java]: It's done

**Review by mekya — APPROVED** [review]

File: src/main/java/io/antmedia/console/rest/CommonRestService.java (hunk 1 above)
Comment (mekya) [reply src/main/java/io/antmedia/console/rest/CommonRestService.java]: Thank you for the update

**Comment by mekya** [comment]
Thank you for the PR. It's merged. If there was a some simple unit test codes, it would be better.

**Pull request merged**